test/reactionsystem_structure/reactionsystem.jl

```diff
@@ -700,3 +698,16 @@ let
     @test rx == rx2
     @test hash(rx) == hash(rx2)
 end
+
+### Additional Unsorted Tests ###
+
+rn = @reaction_network begin k, X --> 0 end
```

Comment:
Put these is their own let block for scoping.

Reply:
done